snap/snapcraft.yaml

```diff
@@ -39,11 +40,11 @@ parts:
       snapdaemon.sh: usr/bin/snapdaemon
 
 plugs:
-   etc-auto-cpufreq-conf:
+  etc-auto-cpufreq-conf:
```

Comment:
I asked this already mentioned multiple times #526, please do not add any characters that are needed. Please remove the empty white space on this line. Depending on file extension this can even break things.

Reply:
done

Reply:
Did you push the changes because I still see it how it was?